memtier_benchmark.cpp

```diff
@@ -235,7 +235,7 @@ static void config_init_defaults(struct benchmark_config *cfg)
     if (!cfg->taskset.is_defined() && cfg->cpu_split)
         cfg->taskset = config_cpu_list(get_nprocs());
     if (!cfg->ratio.is_defined())
-        cfg->ratio = config_ratio("1:10");
+        cfg->ratio = cfg->crc_verify ? config_ratio("1:0") : config_ratio("1:10");
```

Comment:
why dont we do the same .. specify ratio

Reply:
As we discussed we currently support only 1:0 ratio and then verifcation of the data

Reply:
@yael-lb  can you explain why for those who weren't part of the discussion.

Reply:
@adir-lb it can be extended to support different ratios and different pattern. It would take more time. At the moment, as I understood, we need something that works.